examples/step-37/doc/results.dox

```diff
@@ -11,122 +11,193 @@ solution through both isocontours and volume rendering:
-Time solve (5 iterations)  (CPU/wall) 0s/0.000926s
+Total setup time               (wall) 0.00114608s
+Time solve (6 iterations)  (CPU/wall) 0.000935s/0.000934839s
 
 Cycle 2
 Number of degrees of freedom: 1089
-System matrix memory consumption:     0.05286 MB.
-Multigrid objects memory consumption: 0.1581 MB.
-Total setup time               (wall) 0.003045s
-Time solve (6 iterations)  (CPU/wall) 0.012s/0.003393s
+Total setup time               (wall) 0.00244665s
+Time solve (6 iterations)  (CPU/wall) 0.00207s/0.002069s
 
 Cycle 3
 Number of degrees of freedom: 4225
-System matrix memory consumption:     0.1957 MB.
-Multigrid objects memory consumption: 0.5228 MB.
-Total setup time               (wall) 0.008561s
-Time solve (6 iterations)  (CPU/wall) 0.02s/0.01133s
+Total setup time               (wall) 0.00678205s
+Time solve (6 iterations)  (CPU/wall) 0.005616s/0.00561595s
 
 Cycle 4
 Number of degrees of freedom: 16641
-System matrix memory consumption:     0.7343 MB.
-Multigrid objects memory consumption: 1.925 MB.
-Total setup time               (wall) 0.02938s
-Time solve (6 iterations)  (CPU/wall) 0.068s/0.03312s
+Total setup time               (wall) 0.0241671s
+Time solve (6 iterations)  (CPU/wall) 0.019543s/0.0195441s
 
 Cycle 5
 Number of degrees of freedom: 66049
-System matrix memory consumption:     2.856 MB.
-Multigrid objects memory consumption: 7.435 MB.
-Total setup time               (wall) 0.1128s
-Time solve (6 iterations)  (CPU/wall) 0.228s/0.09577s
+Total setup time               (wall) 0.0967851s
+Time solve (6 iterations)  (CPU/wall) 0.07457s/0.0745709s
 
 Cycle 6
 Number of degrees of freedom: 263169
-System matrix memory consumption:     11.28 MB.
-Multigrid objects memory consumption: 29.3 MB.
-Total setup time               (wall) 0.4553s
-Time solve (6 iterations)  (CPU/wall) 1.272s/0.3955s
+Total setup time               (wall) 0.346374s
+Time solve (6 iterations)  (CPU/wall) 0.260042s/0.265033s
 @endcode
 
 As in step-16, we see that the number of CG iterations remains constant with
-increasing number of degrees of freedom. We can also see that the various
-objects we have to store for the multigrid method on the individual levels of
-our mesh together make up more than twice as much as the matrix on the finest
-level. For the present example, about half the memory consumption of the
-multigrid objects are the level transfer matrices, and the other half is
-consumed by the matrix-free objects (and there, mainly the indices and the
-variable coefficient).
+increasing number of degrees of freedom. A constant number of iterations
+(together with optimal computational properties) means that the computing time
+approximately quadruples as the problem size quadruples from one cycle to the
+next. A look at the memory consumption reveals that the code is also very
+efficient in terms of storage. Around 2-4 million degrees of freedom fit into
+1 GB of memory. An interesting fact is that solving one linear system is
```

Comment:
I removed the memory consumption output in here since it is not very accurate. I could have added an output of the memory stats but it does not really help in understanding the program. I will reword.